site/content/en/docs/Guides/player-tracking.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Player Tracking"
+linkTitle: "Player Tracking"
+date: 2020-05-19
+weight: 30
+description: >
+  Track player connections, disconnections, counts and capacities through the Agones SDK
+---
+
+{{< alpha title="Player Tracking" gate="PlayerTracking" >}}
+
+## Managing GameServer Capacities
+
+To track your `GameServer` current player capacity, Agones gives you the ability to both set an initial capacity at
+`GameServer` creation, as well be able to change it during the lifecycle of the `GameServer` through the Agones SDK.
+
+To set the initial capacity, you can do so via `GameServer.Spec.Players.InitialCapacity` like so:
+
+```yaml
+apiVersion: "agones.dev/v1"
+kind: GameServer
+metadata:  
+  name: "gs-example"
+spec:
+  # ...
+  players:
+    # set this GameServer's initial player capacity to 10
+    initialCapacity: 10
+```
+
+From there, if you need to change the capacity of the GameSever as gameplay is in progress, you can also do so via 
+[`SDK.Alpha().SetPlayerCapacity(count)`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-setplayercapacity-count" >}}) 
+
+The current player capacity is represented in `GameServer.Status.Players.Capacity` resource value.
+
+We can see this in action, when we look at the Status section of a GameServer resource
+, wherein the capacity has been set to 20:
+
+```
+...
+Status:
+  Address:    14.81.195.72
+  Node Name:  gke-test-cluster-default-6cd0ba67-1mps
+  Players:
+    Capacity:  20
+    Count:     0
+    Ids:       <nil>
+  Ports:
+    Name:          gameport
+    Port:          7983
+  Reserved Until:  <nil>
+  State:           Ready
+```
+
+From the SDK, the game server binary can also retrieve the current player capacity 
+via [`SDK.Alpha().GetPlayerCapacity()`]({{< ref "/docs/Guides/Client SDKs/_index.md#alpha-getplayercapacity" >}}).
+
```

Comment:
Making a note here, to make sure I make the change:
https://github.com/googleforgames/agones/pull/1570#discussion_r427132641

> It might be worth clarifying that this capacity is not enforced as that is not a responsibility of Agones

Reply:
@steven-supersolid added notes here regarding both capacity changes, and also connect/disconnect.

Reply:
Great. This feature is really cool BTW and should enable us to remove some code!